index.html

```diff
@@ -8,7 +8,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title>LinuxChix India</title>
     <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
```

Comment:
Both sides have their own advantage, usually developers keep `user-scalable=no` because it helps the mobile browsers to fire click events faster. There's no denying that `user-scalable=no` is generally considered bad for UX ([Read](https://ux.stackexchange.com/questions/80487/revisiting-user-scalable-no)). So, it would have to be a choice as per your requirements, better UX for your mobile users or faster click events for your mobile users.